site/app/templates/grading/electronic/Status.twig

```diff
@@ -1,164 +1,32 @@
+<head>
+    <!-- Plotly.js -->
+    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
```

Comment:
Could we add this JS file locally through `addInternalJS` instead of fetching it from elsewhere?